task/build-image-manifest/0.1/kustomization.yaml

```diff
@@ -1,5 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-
-resources:
-- build-image-manifest.yaml
```

Comment:
These deleted files seem unrelated to this PR

Reply:
We don't need these files. There has been a PR open to deleted them for weeks.